### PR TITLE
update to use safecache types for controller

### PR DESCRIPTION
**Pull request by ChrisMacNaughton**

### CODE REVIEW

**Comment by ChrisMacNaughton** [comment]
The cores and memory constraints have to be removed because they conflict with the instance-type constraint

**Comment by gnuoy** [comment]
LGTM

File: juju-openstack-controller-example.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ set -ux
 : ${CONTROLLER_NAME:="${OS_PROJECT_NAME}-${CLOUD_NAME}"}
 : ${MODEL_NAME:="${OS_PROJECT_NAME:0:12}"}
 : ${NETWORK_ID:=$(openstack network list | awk "/${OS_PROJECT_NAME}_admin_net/"'{ print $2 }')}
-: ${BOOTSTRAP_CONSTRAINTS:="virt-type=kvm cores=4 mem=8G"}
+: ${BOOTSTRAP_CONSTRAINTS:="virt-type=kvm instance-type=m1.safecache.large"}
```
Comment (ryan-beisner) [inline juju-openstack-controller-example.sh:15]: I'm concerned that this will fail if that very-specifically-named flavor doesn't exist.

**Review by ryan-beisner — CHANGES_REQUESTED** [review]
-1 from me.

**Comment by lourot** [comment]
@ChrisMacNaughton do we still want this?

**Comment by ChrisMacNaughton** [comment]
Nah, as we're not actually keeping controllers around anymore :)

**Pull request closed without merging**